File: desafio_obj_e_modulos/js/services/shippingService.js


export function  ShippingService(order){
    if (order < 100){
        return  20.0;
    }
    if(order == 100 || order <= 200){
        return 12.0;
    }
    if (order > 100){
     return 0.0
    }
    }

File: desafio_Componentes/desafio/src/main/java/com/desafiocomponentes/desafio/services/ShippingService.java
package com.desafiocomponentes.desafio.services;

import com.desafiocomponentes.desafio.entities.Order;
import org.springframework.stereotype.Service;

@Service
public class ShippingService {
    public double shipment(Order order) {
        if (order.getBasic() < 100.00) {
            return 20.00;
        }
        if (order.getBasic() <= 200.00) {
            return 12.00;
        }
        else {
            return 0.00;
        }
    }

}
